docs/content/v0.20/packaging.md

```diff
@@ -21,13 +62,15 @@ ends with `/github.com/kubestellar/kubestellar`.
 
 ### Derived files
 
-Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub. A contributor can use the individual `make` commands mentioned below to update subsets, or use the following command to update them all.
+Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub.
+
+Some of these derived files are derived by standard generators from the Kubernetes milieu. A contributor can use the following command to make all of those, or use the individual `make` commands described in the following subsubsections to update particular subsets.
 
 ```shell
 make all-generated
 ```
 
-The following command, checked in CI, checks whether all the derived files have been correctly derived. It must be invoked in a state where the `git status` is clean, or at least the dirty files are irrelevant; the current commit is what is checked.
+The following command, checked in CI, checks whether all those derived files have been correctly derived. It must be invoked in a state where the `git status` is clean, or at least the dirty files are irrelevant; the current commit is what is checked.
```

Comment:
I thought it also generate them not just "check"

Reply:
Noted.